Training/TPU-Trillium/Llama3-70B-PyTorch/env.sh

```diff
@@ -6,6 +6,7 @@ XLA_HLO_DEBUG=1
 PROFILE_EPOCH=0
 PROFILE_STEP=3
 PROFILE_DURATION_MS=120000
+PROFILE_LOGDIR=/tmp/home/profile
```

Comment:
@zpcore why do we have PROFILE_DIR [here](https://github.com/gclouduniverse/reproducibility/blob/main/Training/TPU-Trillium/Diffusion-2-PyTorch/env.sh)?

Reply:
The env only apply to stable diffusion as we set here: https://github.com/pytorch-tpu/diffusers/blob/58f2d6a31473bd9737726123df9cb7ecf5c58c7f/examples/text_to_image/train_text_to_image_xla.py#L30